docs/trees/astexplorer.md

```diff
@@ -1,10 +1,10 @@
 ---
 id: astexplorer
-title: AST Explorer
+title: Scalameta AST Explorer
 ---
 
 A great way to learn about the structure of Scalameta syntax trees is
-[AST explorer](https://astexplorer.net/#/gist/ec56167ffafb20cbd8d68f24a37043a9/677e43f3adb93db8513dbe4e2c868dd4f78df4b3).
+[Scalameta AST explorer](https://github.com/keynmol/scalameta-ast-explorer).
```

Comment:
Tomasz and I are currently talking about moving the explorer under scalameta org, so let's hold off until that's done – the static app will be hosted under scalameta.org domain

Reply:
It's done!

Live link at https://scalameta.org/ast-explorer/
Github at https://github.com/scalameta/ast-explorer/

Reply:
Yep, I think the other one doesn't currently work 🤔 

https://github.com/scalameta/scalameta/pull/4219